data/blog/bitcoin-infrastructure-this-decade.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Looking Ahead: Bitcoin Infrastructure This Decade"
+date: '2023-09-20'
+tags: ['OpenSats', 'bitcoin', 'infrastructure', 'research']
+draft: false
+authors: ['niftynei']
+images: ['/static/images/blog/09-nifty-infra.jpg']
+summary: "What will the next decade of bitcoin infrastructure building bring?"
+---
+
+OpenSats aims to be a low-time preference, long-term thinking organization. One good example of this is our recent announcement of the LTS (Long Term Support) program for funding core contributors with a track record of solid contributions.
+
+As an organization, we aim to focus our funding and efforts on open-source code and education projects that will prove to provide long-term value for bitcoin as infrastructure. As such, we prioritize funds to builders and experts who are focused on improving the resilience, accessibility, and reach of bitcoin through important and maybe underfunded work on core technology in the ecosystem.
+
+As infrastructure, it's important that the tech we build supports bitcoin's core values: self-sovereignty through the preservation of anonymity, accessibility, and decentralization. Together, these values build us a system where trustless financial transactions without intermediaries are a touchable reality for all, not an imagined ideal.
+
+Given these goals, what does a bitcoin infrastructure look like a decade from now?
+
+- **Mining**, particularly transaction inclusion decisions, and block construction are more distributed, returning true value to the mempool as a necessary gossip protocol for the network.
+- **Mempool:** Improvements to the mempool (clusters, package relay, ephemeral anchors) help bitcoin scale through more robust and miner-compatible open infrastructure, providing every bitcoin node with reasonable access to competitive information and rate-setting in the global fee market.
+- **Layer 2:** A robust and interoperable set of options for "second layer" transaction ledgers, which offer bitcoiners the opportunity to stack and transact at different scales and timeline tradeoffs.
+- **Self-custody:** Accessible and understandable self-custodial tools and education, particularly opportunities for increasing the availability of self-custodianship of bitcoin in layer-2s.
```

Comment:
bitcoin lowercase seems all good here because line 22 is referring to custody of the the "coin" itself, "bitcoin."